hanadb_exporter/secret_manager.py

```diff
@@ -0,0 +1,32 @@
+import boto3
+import requests
+import json
+from botocore.exceptions import ClientError
+
+def get_db_credentials(secret_name):
+    ec2_info_url = "http://169.254.169.254/latest/dynamic/instance-identity/document"
+    ec2_info = requests.get(ec2_info_url).json()
```

Comment:
I miss some error handling here

Reply:
ok I will check that

Reply:
done https://github.com/SUSE/hanadb_exporter/pull/97